ddtrace/contrib/requests/connection.py

```diff
@@ -81,5 +85,8 @@ def _wrap_request(func, instance, args, kwargs):
                     span.set_tag(http.STATUS_CODE, response.status_code)
                     # `span.error` must be an integer
                     span.error = int(500 <= response.status_code)
+                    # Storing response headers in the span. Note that response.headers is not a dict, but an iterable
+                    # requests custom structure, that we convert to a dict
+                    store_response_headers(dict(getattr(response, 'headers', {}).items()), span, config.requests)
```

Comment:
I think I'd prefer the `dict(getattr(response, 'headers', {}).items())` part to not be inline here, makes this line a little difficult to read.

Reply:
Also, can move the `Note that ...` to start on it's own line.